### DISCUSSION TITLE
types of voronoi

### DISCUSSION THREAD

**Discussion by Rio2109** [neperfepx/neper#828] [Q&A] [answered]
How to generate tessellation using inverse voronoi, multiplicative or linear voronoi in NEPER software? by default it uses power voronoi algorithm but how to implement other types of voronoi?

**Comment by rquey** [comment, marked as answer]
Neper works with Voronoi and Laguerre (power-Voronoi) tessellations. Tessellations that generate non-convex cells are not supported and would not fit well in the current framework.
